InformationScripting/src/graph/Property.cpp

```diff
@@ -33,5 +33,10 @@ boost::python::object pythonObject(const Property& p)
 	return p.data_->pythonObject();
 }
 
+bool Property::operator==(const Property& other) const
```

Comment:
Yes, you had pretty much an identical point to what I say here, in an answer to the hashing question I had yesterday.